vta/tests/python/integration/test_benchmark_topi_conv2d.py

```diff
@@ -24,10 +30,30 @@
 import topi.testing
 import vta
 import vta.testing
-import numpy as np
-
-Workload = vta.top.vta_conv2d.Workload
-
+from vta.testing import simulator
+
+Workload = namedtuple("Conv2DWorkload",
+                      ['batch', 'height', 'width', 'in_filter', 'out_filter',
+                       'hkernel', 'wkernel', 'hpad', 'wpad', 'hstride', 'wstride'])
+
+# ResNet18 workloads
```

Comment:
Can we simplify this by using a task extraction style instead of hardcoded workloads?

Reply:
I believe we should, but it will be done as part of another PR